cmd/oras/root/manifest/fetch.go

```diff
@@ -74,8 +74,10 @@ Example - Fetch raw manifest from an OCI layout archive file 'layout.tar':
 			switch {
 			case opts.outputPath == "-" && opts.Template != "":
 				return fmt.Errorf("`--output -` cannot be used with `--format %s` at the same time", opts.Template)
-			case opts.OutputDescriptor && opts.outputPath == "-":
+			case opts.outputPath == "-" && opts.OutputDescriptor:
 				return fmt.Errorf("`--descriptor` cannot be used with `--output -` at the same time")
+			case opts.outputPath != "" && opts.Pretty.Pretty:
```

Comment:
```suggestion
			case opts.outputPath != "" && opts.outputPath != "-"  && opts.Pretty.Pretty:
```

Reply:
Also need a comment in the case block, e.g. `--pretty only effects stdout output`